# These sequences of ArmStates make up the trajectory.
ArmState[] arm
duration[] timing

# These record the dominant reference frames for the right and left
# arms.
uint8 refFrame
Landmark refFrameLandmark
